src/app/core/ui/ui/ui.component.html

```diff
@@ -52,14 +53,16 @@
     <app-language-select
       *ngIf="siteSettings.displayLanguageSelect"
     ></app-language-select>
+
+    <app-demo-assistant-button></app-demo-assistant-button>
   </div>
 </mat-toolbar>
 
 <!-- MAIN NAVIGATION + CONTENT -->
 <mat-sidenav-container (backdropClick)="closeSidenavOnMobile()" autosize>
   <mat-sidenav
     #sideNav
-    *ngIf="isLoggedIn()"
+    *ngIf="isLoggedIn | async"
```

Comment:
somehow this is not hiding it during choose-use-case anymore for me right now ...

Reply:
I found the issue. Earlier, when things were working, we mistakenly imported merge from hammerjs. At that time, the detectConfigReadyState() method wasn’t returning anything, so the navigation remained hidden until the setup demo process was completed, and it started returning true.

now, however, we’ve correctly switched to using merge from rxjs. In this setup, detectConfigReadyState() returns true as either a config is present or the sync is complete (which indicates that there’s no existing config on the server). Because of this, the navigation is shown immediately after the system initializes, even when the user is just choosing a use case.
That’s why the navigation isn’t being hidden anymore.